<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>361</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>646</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>259</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Body type</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="c_typeCar">
      <item>
       <property name="text">
        <string>All types</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sedan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pickup</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hatchback</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Universal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Compartment</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Minivan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Crossover</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>259</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Brand</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="c_brandCar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <item>
       <property name="text">
        <string>All brands</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Audi</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Skoda</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Daewoo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>BMW</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mercedes-Benz</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ford</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Toyota</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Volkswagen</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>259</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Graduation year</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>125</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>From</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>125</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>To</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="l_fromYear">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>125</width>
          <height>28</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="l_toYear">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>125</width>
          <height>28</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>259</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Price ($)</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>125</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>From</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>125</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>To</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="l_fromPrice">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>125</width>
          <height>28</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="l_toPrice">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>125</width>
          <height>28</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="b_search">
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>c_typeCar</tabstop>
  <tabstop>c_brandCar</tabstop>
  <tabstop>l_fromYear</tabstop>
  <tabstop>l_fromPrice</tabstop>
  <tabstop>l_toPrice</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
